.github/workflows/build.yml

```
@@ -242,10 +242,10 @@ jobs:
           target: release-light
           arch: ${{ matrix.arch }}
           tags: |
-            docker.io/hashicorp/${{ env.REPO_NAME }}:light
+            ${{ steps.get-product-version.output.prerelease-product-version == "" && format("docker.io/hashicorp/{0}:light", env.REPO_NAME) }}
```

```suggestion
            ${{ steps.get-product-version.output.prerelease-product-version == '' && format('docker.io/hashicorp/{0}:light', env.REPO_NAME) }}
```

One day I'll remember to use single quotes in GHA templates